sympy/calculus/util.py

```diff
@@ -1348,20 +1349,66 @@ def __pow__(self, other):
                         return AccumBounds(self.min**other, self.max**other)
 
                 num, den = other.as_numer_denom()
```

Comment:
At this point, `other` is a number but not an integer and `self.min` is not non-negative. I think that an unevaluated `Pow` should be returned.

Reply:
I might be missing something but at the outset we test `other.is_Integer or self.min.is_extended_nonnegative` so the min *is* non-negative.